<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DriverTestPage</class>
 <widget class="QWidget" name="DriverTestPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel
{
	color: white;
	border: 2px groove, gray;
    border-color: gray;
	font: 20px, &quot;SimHei&quot;;	
}

QWidget
{
	background-color: black;
}

QPushButton
{
	font: 20px, &quot;SimHei&quot;;
    color: black;
	background-color: rgb(250, 250, 250);
	border-radius: 5px;
}
</string>
  </property>
  <widget class="QLabel" name="label2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>105</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Level1">
   <property name="geometry">
    <rect>
     <x>161</x>
     <y>134</y>
     <width>399</width>
     <height>60</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>161</x>
     <y>105</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label4">
   <property name="geometry">
    <rect>
     <x>768</x>
     <y>105</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Level">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>134</y>
     <width>130</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>司控器级位</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_traction1">
   <property name="geometry">
    <rect>
     <x>161</x>
     <y>221</y>
     <width>787</width>
     <height>60</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label3">
   <property name="geometry">
    <rect>
     <x>559</x>
     <y>105</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_traction">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>221</y>
     <width>130</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>    牵引
（CCU-&gt;BCU）</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>105</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>车号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Level2">
   <property name="geometry">
    <rect>
     <x>559</x>
     <y>134</y>
     <width>389</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_brake">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>280</y>
     <width>130</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>    制动
（CCU-&gt;BCU）</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_brake1">
   <property name="geometry">
    <rect>
     <x>161</x>
     <y>280</y>
     <width>787</width>
     <height>60</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_brakePercent1">
   <property name="geometry">
    <rect>
     <x>161</x>
     <y>193</y>
     <width>787</width>
     <height>30</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_brakePercent">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>193</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>制动力百分比</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>490</y>
     <width>98</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
